CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# Code owners for the Timed backend. We include our backend dev team here.
+# Since this is a split project (backend/frontend) it's rather simple
+* @adfinis-sygroup/dev/backend
```

Comment:
Does the slashed notion have any functional meaning to it? Never seen that before

Reply:
That's there to reference (nested) teams

Reply:
Ok I can't see such a nested example in [the docs](https://docs.github.com/en/enterprise/2.15/user/articles/about-code-owners) but fine with me.

Reply:
Yeah you're right - nested teams are still referenced via `org/team-name` without nesting. Changed the file accordingly

Reply:
(... and renamed the teams as well for consistency)